Katello/katello
# Fixes  #35220 - CV Version Compare - add support for different content types

#### What are the changes introduced in this pull request?
Expanded CVVersionCompare to include other content types outside of just Packages
#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?
1. Select two version on the Content View Versions page
2. Hit "Compare" button
3. Make sure different content type tabs are rendered in the TableWrapper
4. Change the versions in the dropdowns
5. Ensure that the table is updates with new types of content types
